src/sentry/static/sentry/app/views/organizationEventsV2/tags.tsx

```diff
@@ -4,27 +4,45 @@ import styled from 'react-emotion';
 import {isEqual, omit} from 'lodash';
 import * as Sentry from '@sentry/browser';
 
+import {Client} from 'app/api';
 import SentryTypes from 'app/sentryTypes';
 import Placeholder from 'app/components/placeholder';
 import TagDistributionMeter from 'app/components/tagDistributionMeter';
 import withApi from 'app/utils/withApi';
+import {Organization, EventView} from 'app/types';
+import {ReactRouterLocation} from 'app/types/reactRouter';
+
 import {
   fetchTagDistribution,
   fetchTotalCount,
   getEventTagSearchUrl,
   getQuery,
+  Tag,
+  TagTopValue,
 } from './utils';
 import {MODAL_QUERY_KEYS} from './data';
 
-class Tags extends React.Component {
-  static propTypes = {
+type Props = {
+  api: Client;
+  organization: Organization;
+  view: EventView;
+  location: ReactRouterLocation;
+};
+
+type State = {
+  tags: {[key: string]: Tag};
+  totalValues: null | number;
+};
+
+class Tags extends React.Component<Props, State> {
+  static propTypes: any = {
```

Comment:
Is it that hard to get these to match? what is it complaining about?

Reply:
because of ` Type 'Validator<object>' is not assignable to type 'Validator<Client>'.`